traitsui/qt4/table_editor.py

```diff
@@ -1246,7 +1246,7 @@ def _update_header_sizing(self):
             hheader.setStretchLastSection(True)
         else:
             # hheader.setSectionResizeMode(
-            #     stretchable_columns[-1], QtGui.QHeaderView.Stretch)
+            #     stretchable_columns[-1], QtGui.QHeaderView.ResizeMode.Stretch)
```

Comment:
just pointing out the commented code

Reply:
OK with changes to commented out code, since that code may eventually be not commented out.